cmd/agent/subcommands/diagnose/command.go

```diff
@@ -172,10 +172,27 @@ This command print the inventory-agent metadata payload. This payload is used by
 			)
 		},
 	}
+
+	payloadInventoriesHostCmd := &cobra.Command{
+		Use:   "inventory-host",
+		Short: "Print the Inventory host metadata payload.",
+		Long: `
+This command print the inventory-host metadata payload. This payload is used by the 'inventories/sql' product.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliParams.payloadName = "inventory-host"
```

Comment:
Note: It is unexpected to have payloadName in cliParams as it doesn't come from the CLI.

Reply:
Yes, this will be fixed in another PR.